package/Dockerfile

```diff
@@ -10,12 +10,12 @@ RUN zypper -n rm container-suseconnect && \
 
 WORKDIR /var/lib/harvester/harvester
 
-ENV HARVESTER_UI_VERSION release-harvester-v1.4
+ENV HARVESTER_UI_VERSION v1.4.2
```

Comment:
Is the original `release-harvester-v1.4` meant for 1.4-head build?

Reply:
Yes, it's UI's release branch https://github.com/harvester/dashboard/tree/release-harvester-v1.4

Reply:
yep